content/docs/reference/state.md

```diff
@@ -1,140 +1,307 @@
 ---
 title: State and Backends
+description: Learn about how Pulumi handles your infrastructure state files and supported backend options for these state files.
+keywords:
+ - IaC
+ - infrastructure state
+ - state backend
 expanded_url: /docs/reference/concepts/
 menu:
   reference:
     parent: concepts
     weight: 7
 ---
 
-Pulumi stores its own copy of the current state of your infrastructure. This is often simply called _state_, and is
-stored in transactional snapshots we call _checkpoints_. A checkpoint is recorded by Pulumi at various points so that
-it can operate reliably, whether that means diffing goal state versus current state during an update, recovering from
-failure, or destroying resources accurately to clean up afterwards, for example. Because state is critical to how Pulumi
-operates, this page gives an overview of the options.
+## State
+
+Pulumi stores its own copy of the current state of your infrastructure. This is
+often simply called _state_, and is stored in transactional snapshots we call
+_checkpoints_. A _checkpoint_ is recorded by Pulumi at various points so that
+it can operate reliably---whether that means diffing goal state versus
+current state during an update, recovering from failure, or destroying
+resources accurately to clean up afterwards. Because _state_ is critical to how
+Pulumi operates, we'll cover a few of the state backend options on this page.
 
 ## Backends
 
-Pulumi supports multiple so-called _backends_ for storing this state. There are two primary kinds:
+Pulumi supports multiple  _backends_ for storing your infrastructure
+state:
+
+- The `pulumi.com` backend
+- A self-managed backend, either stored locally on your filesystem or remotely using
+a cloud storage service
+
+
+### pulumi.com backend
+
+The `pulumi.com` backend requires no additional configuration after
+[installing the CLI](/docs/reference/install). By default, the CLI
+uses a web backend hosted at [app.pulumi.com](https://app.pulumi.com).
+
+Pulumi offers this backend as a free service with
+[advanced tiers](https://www.pulumi.com/pricing/) for team and
+enterprise features. Using the [`pulumi.com` backend](#pulumi-backend-features)
+and the CLI together provides a great combination of usability, safety, and
+security for most users.
+
+However, if you would prefer to manage this yourself, you can do so by opting
+into the filesystem or cloud storage backend for more flexibility if you need
+it.
```

Comment:
Suggestion: However, if you would prefer to manage this yourself, you can do so by opting
into the filesystem or cloud storage backend. ~~for more flexibility if you need
it.~~